src/components/_shared/TabBar/TabBar.tsx

```diff
@@ -1,36 +1,54 @@
-import React from 'react'
-import PropTypes from 'prop-types'
+import React, { useRef } from 'react'
+import { uniqueId } from 'lodash'
+import * as colors from 'global_styles/colors'
 import styles from './TabBar.module.sass'
 
+function useId(): string {
+  const { current: id } = useRef(uniqueId())
+  return id
+}
+
+interface Props<T> {
```

Comment:
Maybe I don't quite get it, but a `Tab` is not actually a generic, or is it? In the storybook you describe it as always having `{ value: Number, display: String }`, so why not define it as a type?